examples/speech_to_text.v1.js

```diff
@@ -19,19 +27,20 @@ var recognizeStream = speechToText.createRecognizeStream(params);
 // pipe in some audio
 fs.createReadStream(__dirname + '/resources/speech.wav').pipe(recognizeStream);
 
-// and pipe out the transcription
+/*
+// pipe out the transcription to a file
 recognizeStream.pipe(fs.createWriteStream('transcription.txt'));
 
-// listen for 'data' events for just the final text
-// listen for 'results' events to get the raw JSON with interim results, timings, etc.
+// get strings instead of Buffers from `data` events
+// only do this when objectMode is `false`
+recognizeStream.setEncoding('utf8');
+*/
 
-recognizeStream.setEncoding('utf8'); // to get strings instead of Buffers from `data` events
+recognizeStream.on('data', function(event) { onEvent('Data:', event); });
```

Comment:
I like this approach, Dustin.  At a minimum, I think it's cleaner and easier to follow at first glance.  Thanks!

Reply:
Thanks - I copied that from your code haha

Reply:
You know, I thought it looked familiar... :-)